test/blocks/gist/gist.test.js

```diff
@@ -6,10 +7,43 @@ document.body.innerHTML = await readFile({ path: './mocks/body.html' });
 const { default: init } = await import('../../../libs/blocks/gist/gist.js');
 
 describe('adobetv autoblock', () => {
-  it('creates iframe video block', async () => {
+  let createElementStub;
+  let originalCreateElement;
+
+  beforeEach(() => {
+    originalCreateElement = document.createElement;
+
+    createElementStub = sinon.stub(document, 'createElement').callsFake((tagName) => {
+      if (tagName === 'script') {
```

Comment:
Earlier returns are preferred
```suggestion
      if (tagName !=='script') return
```